src/lib/asn1/ASN1.h

```diff
@@ -186,6 +186,7 @@ class DLL_EXPORT ASN1Writer
     ASN1_ERROR EndEncapsulatedType(void);
     ASN1_ERROR PutValue(uint8_t cls, uint32_t tag, bool isConstructed, const uint8_t * val, uint16_t valLen);
     ASN1_ERROR PutValue(uint8_t cls, uint32_t tag, bool isConstructed, chip::TLV::TLVReader & val);
+    ASN1_ERROR PutRaw(const uint8_t * val, uint16_t valLen);
```

Comment:
This method duplicates PutConstructedType()

Reply:
Oops, removed PutRaw() and updated generator to use PutConstructedType()